slash/log.py

```diff
@@ -189,7 +199,7 @@ def _get_silenced_logs_context(self):
             return ExitStack()
         return SilencedLoggersHandler(config.root.log.silence_loggers).applicationbound()
 
-    def _get_file_log_handler(self, subpath, symlink, bubble=False, filter=None):
+    def _get_file_log_handler(self, subpath, symlink, bubble=False, filter=None, use_compression=False, cyclic=False):
```

Comment:
The term is "rotating", not "cyclic". Cyclic means that old log *lines* get run over by new lines...